oktoberfest/utils/config.py

```diff
@@ -11,7 +13,7 @@ def __init__(self):
         """Initialize config file data."""
         self.data = {}
 
-    def read(self, config_path: str):
+    def read(self, config_path: Union[str, Path]):
```

Comment:
you need to handle this, i.e. make sure it is a Path object before reading from the config path to make it platform independent!